ui/framework/src/ui-framework/frontstage/FrontstageManager.ts

```diff
@@ -545,6 +585,7 @@ export class FrontstageManager {
    * @returns Top-most modal Frontstage, or undefined if there is none.
    */
   public static get activeModalFrontstage(): ModalFrontstageInfo | undefined {
+    // istanbul ignore else

```

Comment:
Why did this ignore need to be added?

Reply:
Can't remember why I added it.... but its gone now.